1_fetch/src/fetch_data.R

```diff
@@ -1,15 +1,7 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
```

Comment:
One thing -- not all of these library are needed to run `fetch_Data()`. This is a bit overkill for this example, but is good practice - can you move the library calls so that each `{X_phase}/src/{script}.R` file loads exactly those libraries needed within that script?

Reply:
I've addressed this; thanks.

Reply:
^ I noted in our ongoing chat, but I'm retracting this comment b/c I had my scipiper hat on, and for `targets` the best practice is to load all libraries in the `_targets.R` file, either defined in `tar_option_set()` or as a package required for a specific target listed under the `packages = c()` argument to `tar_target()`